src/modules/emote_socket_listeners/youtube.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import socketClient, {EventNames} from '../../socket-client.js';
+import watcher from '../../watcher.js';
+import channelEmotes from '../emotes/channel-emotes.js';
+import YoutubeHiddenMessage from '../../common/components/YoutubeEphemeralMessage.jsx';
+import {getLiveChat} from '../../utils/youtube.js';
+import {validChannelDestination} from './common.js';
+
+async function sendEphemeralMessage(message) {
+  const items = document.querySelector('.yt-live-chat-item-list-renderer > #items');
+  if (items == null) {
+    return;
+  }
+  const visibleItems = getLiveChat()?.visibleItems;
+  if (visibleItems == null) {
+    return;
+  }
+  const bttvMessageContainer = document.createElement('div');
+  bttvMessageContainer.id = 'bttv-chat-admin-message';
+  const root = ReactDOM.createRoot(bttvMessageContainer);
+  /* eslint-disable-next-line react/jsx-filename-extension, react/react-in-jsx-scope */
+  root.render(<YoutubeHiddenMessage message={message} />);
```

Comment:
for twitch we have a util for sending notices to chat. perhaps we need a similar generic util for youtube?

Reply:
I have not found a similar util yet, thus the janky solution. However, I will have another look.

Reply:
i mean to move this to a generic util we can re-use